### DISCUSSION TITLE
Pico theme home page content

### DISCUSSION THREAD

**Discussion by ratulkhan74** [cadentcom/pico-wp-block-theme#27] [General]
@stratofax For our pico theme home page content we need few things I am going to list in the below

1) First we need a catagory that is we have selected already (charity).
2) Then we need a content layout (the design) it could be our custom design on figma or could be a design sample / idea.
- we already researched few design ideas on charity category
- Here is our design samples links:
https://harmony-template.webflow.io/ (This one i liked most than others link this design minimal, clean, simple)

Here is more links that you can also select if you like

https://hapi-128.webflow.io/
https://lovelyacts.webflow.io/
https://empower-template.webflow.io/
https://bracketweb.com/cleenhearts-html/index.html

**Comment by stratofax** [comment]
@ratulkhan74 Some specific implementation questions:

- Where is the sample content stored in the theme directory? Do we need to import it using a plugin?
- How do we ensure that our home page content is used when the user selects the pico-block theme for the first time?

**Reply by ratulkhan74** [reply]
@stratofax here is your question answers:

1) Where is the sample content stored in the theme directory? Do we need to import it using a plugin?
- For the images and icons in png or jpg or svg format also fonts will be stored in the theme assets directory. We don't need any plugins for all of this.

2) How do we ensure that our home page content is used when the user selects the pico-block theme for the first time?
- To ensure that our home page content is used when user active our pico theme we have to create a page name after front-page or home page usally people use the name front page

**Reply by stratofax** [reply]
OK, as long as the sample content is included in the theme files, we can start working on this **after we merge the current PR** (#24) on a new branch.
